extension/js/service_worker/background.ts

```diff
@@ -5,23 +5,29 @@
 import { GoogleOAuth } from '../common/api/authentication/google/google-oauth.js';
 import { Bm, BrowserMsg } from '../common/browser/browser-msg.js';
 import { emailKeyIndex } from '../common/core/common.js';
-import { VERSION } from '../common/core/const.js';
 import { ExpirationCache } from '../common/core/expiration-cache.js';
+import { BgHandlers } from './bg-handlers.js';
 import { Catch } from '../common/platform/catch.js';
 import { ContactStore } from '../common/platform/store/contact-store.js';
-import { GlobalStore, GlobalStoreDict } from '../common/platform/store/global-store.js';
-import { BgHandlers } from './bg-handlers.js';
 import { BgUtils } from './bgutils.js';
-import { injectFcIntoWebmail } from './inject.js';
 import { migrateGlobal, moveContactsToEmailsAndPubkeys, updateOpgpRevocations, updateSearchables, updateX509FingerprintsAndLongids } from './migrations.js';
+import { GlobalStore, GlobalStoreDict } from '../common/platform/store/global-store.js';
+import { VERSION } from '../common/core/const.js';
+import { injectFcIntoWebmail } from './inject.js';
 
-console.info('background_process.js starting');
+console.info('background.js service worker starting');
 
 (async () => {
+  chrome.storage.onChanged.addListener((changes, namespace) => {
+    for (const [key, { oldValue, newValue }] of Object.entries(changes)) {
+      console.log(`Storage key "${key}" in namespace "${namespace}" changed.`, `Old value was "${oldValue}", new value is "${newValue}".`);
+    }
+  });
```

Comment:
this one can be removed too, if you don't use it for debugging

Reply:
Okay